package.json

```diff
@@ -19,20 +20,20 @@
     "let",
     "subject"
   ],
-  "author": {
-    "name": "Gordon L. Hempton",
-    "email": "ghempton@gmail.com"
-  },
+  "author": [
+    "Gordon L. Hempton <ghempton@gmail.com>",
+    "Devin Canterberry <devin.canterberry@outreach.io>"
+  ],
   "license": "ISC",
-  "peerDependencies": {
-    "mocha": ">=2.4.0"
+  "dependencies": {
+    "mocha": "3.5.0"
   },
   "devDependencies": {
-    "chai": "^1.10.0",
-    "mocha": ">=2.4.0",
-    "webpack": "^1.4.15"
-  },
-  "dependencies": {
-    "escape-string-regexp": "^1.0.3"
+    "babel-cli": "6.24.1",
```

Comment:
With `yarn.lock` we don't need exact versions here and is nice to be able to update minors.